runtime/doc/options.txt

```diff
@@ -950,7 +950,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 	error	    Other Error occurred (e.g. try to join last line)
 		    (mostly used in |Normal-mode| or |Cmdline-mode|).
 	esc	    hitting <Esc> in |Normal-mode|.
-	ex	    In |Visual-mode|, hitting |Q| results in an error.
+	ex	    In |Visual-mode|, hitting |gQ| results in an error.
```

Comment:
Unlike `Q`, `gQ` just does nothing in Visual mode and doesn't result in an error, so I'd suggest removing this line.

Reply:
removed in adefbcb8e724a4a917a8505e598ac47164be0858